index.bs

```diff
@@ -5798,6 +5803,109 @@ map between [=user contexts=] and [=screen orientation override=].
 
 ### Commands ### {#module-emulation-commands}
 
+#### The emulation.setDisableScriptingOverride Command ####  {#command-emulation-setDisableScriptingOverride}
+
+The <dfn export for=commands>emulation.setDisableScriptingOverride</dfn> command
+emulates enabling or disabling JavaScript on web pages.
+
+<dl>
+   <dt>Command Type</dt>
+   <dd>
+    <pre class="cddl" data-cddl-module="remote-cddl">
+      emulation.SetDisableScriptingOverride = (
+        method: "emulation.setDisableScriptingOverride",
+        params: emulation.SetDisableScriptingOverrideParameters
+      )
+
+      emulation.SetDisableScriptingOverrideParameters = {
+        disable: true / null,
+        ? contexts: [+browsingContext.BrowsingContext],
+        ? userContexts: [+browser.UserContext],
+      }
+    </pre>
+   </dd>
+   <dt>Result Type</dt>
+   <dd>
+    <code>
+     EmptyResult
+    </code>
+   </dd>
+</dl>
+
+<div algorithm>
+
+To <dfn>get emulated script disabled status</dfn> given |navigables|:
+
+1. For each |navigable| or |navigables|:
+
+   1. If [=disable scripting overrides map=] contains |navigable|, return
```

Comment:
TODO: get  top-level navigable.

Reply:
It makes sense to me that this would work on a per-navigable basis, not just for top-level traversables.

Reply:
what about inheritance? What should it do in case of the emulation is enabled for a child navigable, but then explicitly disabled for the parent? I believe we had more-or-less the same issue with subscriptions and ended up to stick to the top-level traversables for the same reason.

Reply:
All the existing APIs (maybe I miss one?) require a top-level browsing context to be specified. So it would be the first API that would behave differently. Do we really want such a mix?

Reply:
Scripting is specifically tied to a navigable (each navigable has its own global, etc.) so in this case I'm not sure what the problem would be. Arguably there aren't clear use cases for per-navigable overrides but I don't think there's the same level of implementation complexity as with other features.